src/components/table.js

```diff
@@ -394,25 +394,35 @@ GridTable.propTypes = {
   onScroll: PropTypes.func
 }
 
-export const SimpleTable = ({ columns, rows }) => {
+export const SimpleTable = ({ columns, rows, ...props }) => {
   const cellStyles = { paddingTop: '0.25rem', paddingBottom: '0.25rem' }
   return h(Fragment, [
-    div({ style: { display: 'flex' } }, [
+    div({
+      style: { display: 'flex' },
+      role: 'grid',
+      ...props
+    }, [
       _.map(({ key, header, size }) => {
-        return div({ key, style: { ...cellStyles, ...styles.flexCell(size) } }, [header])
+        return div({
+          key,
+          style: { ...cellStyles, ...styles.flexCell(size) },
+          role: 'columnheader'
+        }, [header])
       }, columns)
     ]),
     _.map(([i, row]) => {
       return h(Interactive, {
         key: i,
         as: 'div',
         style: { display: 'flex' }, className: 'table-row',
-        hover: { backgroundColor: colors.light(0.4) }
+        hover: { backgroundColor: colors.light(0.4) },
+        role: 'row'
       }, [
         _.map(({ key, size }) => {
           return div({
             key,
             className: 'table-cell',
+            role: 'gridcell',
```

Comment:
Love these role additions, but as a separate PR so we can contain the new functionality as much as possible.